eatsmart/templates/inspections/establishment_list.html

```diff
@@ -73,15 +73,34 @@
         </form>
         <div class="list-group">
             {% for establishment in establishments %}
-            <a href="{% url "establishment-detail" establishment.pk %}" class="list-group-item">
-                {% if establishment.grade %}<span class="badge">{{ establishment.grade }}</span>{% endif %}
-                <h5 class="list-group-item-heading">{{ establishment|title }}
+            <a href="{% url "establishment-detail" establishment.pk %}" class="list-group-item foodlisting">
+              
+                 <div class="row">
+                    <h5 class="col-xs-9 col-md-10 list-group-item-heading ">{{ establishment|title }}
                     {% if establishment.property_id %}<span class="glyphicon glyphicon-camera"></span>{% endif %}
-                </h5>
-                {% if establishment.insp_date %}<span class="badge-date hidden-xs">{{ establishment.insp_date|date:"M dS Y" }}</span>{% endif %}
-                <p class="list-group-item-text">
-                    <span class='text-muted small'>{{ establishment.address|title }} - {{ establishment.distance.mi|floatformat:"2" }} mi</span>
-                </p>
+                    </h5>
+                    <div class="col-xs-3 col-md-2 small text-muted text-right">{{ establishment.distance.mi|floatformat:"2" }} mi</div>
+                 </div>
+              
+                <div class="row">
+                  <div class="col-xs-5 col-md-4 small text-muted">
+                  {{ establishment.address|title }}
+                   <br />
+                     {% if establishment.grade %}<span class="badge">{{ establishment.grade }}</span>{% endif %}
+                     {% if establishment.insp_date %}<span class="badge-date hidden-xs">{{ establishment.insp_date|date:"M dS Y" }}</span>{% endif %}
+                  </div>
+                  
+                 
+                  <ul class="risk-factor col-xs-7 col-md-8">
```

Comment:
@victor-rocha it wasn't quite that simple because any returns False for zero values, but I have a fix here
https://github.com/codefordurham/food-inspector/pull/163